src/index.ts

```diff
@@ -5,7 +5,11 @@ function load() {
   try {
     return require('../build/Release/mongocrypt.node');
   } catch {
-    return require('../build/Debug/mongocrypt.node');
+    try {
+      return require('../build/Debug/mongocrypt.node');
+    } catch {
+      throw new Error('Could not load the native module mongocrypt.node');
```

Comment:
```suggestion
    } catch (error) {
      throw error;
```

Anything wrong with raising require's own error? then we get the "require stack" information which can help find a deeply nested import:

```
Uncaught Error: Cannot find module '../build/Release/mongocrypt.node'
Require stack:
- /Users/neal/code/drivers/js/mongodb-client-encryption/lib/index.js
- <repl>
    at Module._resolveFilename (node:internal/modules/cjs/loader:1248:15)
    at Module._load (node:internal/modules/cjs/loader:1074:27)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)
    at Module.require (node:internal/modules/cjs/loader:1339:12)
    at require (node:internal/modules/helpers:125:16)
    at load (/Users/neal/code/drivers/js/mongodb-client-encryption/lib/index.js:8:16) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/neal/code/drivers/js/mongodb-client-encryption/lib/index.js',
    '<repl>'
  ]
}
```